apps/api/src/sibyl/jobs/source_imports.py

```
@@ -411,6 +413,17 @@ async def check_duplicate(
     return check_duplicate
 
 
+def _resolve_import_source_uri(source_uri: str) -> str:
+    raw_path = source_uri[7:] if source_uri.startswith("file://") else source_uri
```

Ruff (0.15.13)

[error] Prefer `str.removeprefix()` over conditionally replacing with slice.

Use removeprefix instead of ternary expression conditional upon startswith.

(FURB188)